docs/content/preview/releases/techadvisories/ta-23476.md

```diff
@@ -17,161 +17,105 @@ type: docs
 
 |          Product           |  Affected Versions  |  Related Issues   | Fixed In |
 | :------------------------- | :------------------ | :---------------- | :------- |
-| {{<product "ycql">}}       | {{<release "All">}} | {{<issue 11052>}}, {{<issue 23476>}} | TBD |
+| {{<product "ycql">}}       | {{<release "All">}} | {{<issue 11052>}}, {{<issue 23476>}} | {{<release "2.25.0.0, 2024.1.4.0, 2024.2.1.0, 2.20.9.0">}} |
 
 ## Description
 
-In YCQL, timestamp data inserted with the `currenttimestamp()` function may not be returned for queries with an equality condition on the timestamp. If the timestamp is part of the primary key, this also affects index scans as the table lookup for an index scan internally uses an equality condition to look up the corresponding table rows. The data can still be retrieved as expected by primary key or index-only scan queries with no condition or inequality conditions on the timestamp column.
+In YCQL, timestamp data inserted with the `currenttimestamp()` function or explicit timestamp strings with microseconds precision may not be returned for queries with an equality condition on the timestamp. This can cause discrepancies during queries or index scans on tables where timestamps were part of the primary key. The data can still be retrieved as expected by primary key or index-only scan queries with no condition or inequality conditions on the timestamp column.
```

Comment:
> explicit timestamp strings with microseconds precision
good to provide an example (e.g. 2024-08-26 19:23:38.537281+0000)

Reply:
are you asking to add an example or replace the line as you suggested?

@utkarsh-um-yb

Reply:
Hey @utkarsh-um-yb based on my understanding of your comment, I've added the timestamp example, if you were saying something do let me know I make the changes later on.